src/components/Home.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { showPlanet } from '../redux/home/homeSlice';
+import logo from '../assets/solar-system.png';
+import earth from '../assets/worldwide.png';
+import venus from '../assets/venus.png';
+import mars from '../assets/mars.png';
+import mercury from '../assets/mercury.png';
+import saturn from '../assets/saturn.png';
+import neptune from '../assets/neptune.png';
+import jupiter from '../assets/jupiter.png';
+import uranus from '../assets/uranus.png';
+
+const Home = () => {
+  const planets = useSelector((state) => state.planets.planets);
+
+  const dispatch = useDispatch();
+  const [searchInput, setSearchInput] = useState('');
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setSearchInput(e.target.value);
+  };
+
+  const filter = planets.filter((planet) => planet.name.toLowerCase()
+    .match(searchInput.toLowerCase()));
```

Comment:
- [ ] Filter works perfectly :+1:  but based on the requirement of this project, this was supposed to be achieved on a component that holds **no state**( **Filter** stateless component).